### DISCUSSION TITLE
有个别汉字无法生成，比如“旸”

### DISCUSSION THREAD

**Discussion by KyleYueye** [sjtug/SJTUThesis#617] [提问] [answered]
**问题描述**:
有个别汉字无法生成，比如“旸”，请问如何解决？

**系统信息**:
 
- OS版本(带版本号): Windows
- TeX 发行版(带版本号): 在Overleaf上使用
- 
**日志信息**:
https://pastebin.ubuntu.com/p/ByTWbKdSdc/

**Comment by tanukihee** [comment]
不用 fandol 字体

**Comment by KyleYueye** [comment]
> 不用 fandol 字体

请问具体在哪里改一下？

**Comment by skyzh** [comment]
Duplicate of https://github.com/sjtug/SJTUThesis/issues/533
请参考 https://github.com/sjtug/SJTUThesis/issues/533

**Comment by KyleYueye** [comment]
> Duplicate of #533
> 请参考 #533

解决好了，非常感谢！

**Comment by skyzh** [comment, marked as answer]
在线环境中，参考 https://github.com/sjtug/SJTUThesis/wiki/%E5%9C%A8%E7%BA%BF%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E

离线环境中，可以修改 CTEX 的 fontset。比如：
```
\documentclass[type=master,fontset=ubuntu]{sjtuthesis}
```

可以用的 fontset 包括 `adobe`, `ubuntu`, `windows`, `mac`, `fandol` 等。
